llvm/lib/CodeGen/MachineVerifier.cpp

```diff
@@ -1627,18 +1626,18 @@ void MachineVerifier::verifyPreISelGenericInstruction(const MachineInstr *MI) {
       break;
     }
 
+    if (DstTy.getScalarSizeInBits() < 2) {
+      report("result type must be at least 2 bits wide", MI);
```

Comment:
It looks like most error messages start with a capital letter.
```suggestion
      report("Result type must be at least 2 bits wide", MI);
```

Reply:
The programming guide actually states to use lowercase error messages, but there are many violations